src/storage/flatfile.rs

```diff
@@ -0,0 +1,31 @@
+use super::{Engine, DatabaseError, Column};
+
+use std::io;
+use std::fs::{ OpenOptions,File,create_dir };
```

Comment:
Actually, in `use` statements the `{ }` should touch the things on both sides. And Whitespace after comma please. Like so: `use std::fs::{OpenOptions, File, create_dir};`